extensions/wikia/AnalyticsEngine/AnalyticsProviderKrux.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class AnalyticsProviderKrux implements iAnalyticsProvider {
+
+	private static $siteId = 'JU3_GW1b';
```

Comment:
In single repo we have only one occurrence of this id. I think it's ok because it's a part of request sent to krux for library (it's visible anyway).